### ISSUE TITLE
[css-fonts-4] Animation type of font-size-adjust: none

### ISSUE THREAD

**Issue by cdoublev** [w3c/csswg-drafts#5762]
Hello,

In the [draft of CSS Fonts 4](https://drafts.csswg.org/css-fonts-4/#font-size-adjust-prop), the animation type of `font-size-adjust` is `by computed value type` and the computed value is `a number or the keyword none`.

I believe other specs use `discrete` whenever at least one authorized value prevents interpolation. In Firefox, the animation type from `none` to `number` is discrete. 

To sum up, this is just a suggestion to change `by computed value` to `discrete`, for consistency with other specifications.

EDIT: actually I believe this should be the other way around (any property that can be computed to a value that can be interpolated) should be defined with `by computed value` and not with `discrete`, so I'm closing this issue, sorry.

**Comment by Loirooriol** [comment]
Note that "by computed value" already implies "discrete" when necessary. https://drafts.csswg.org/web-animations-1/#by-computed-value

> If the number of components or the types of corresponding components do not match, or if any component value uses discrete animation and the two corresponding values do not match, then the property values combine as discrete.
